troubleshooting-input-type-file-not-working-on-android.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Input Type File Not Working on Adnroid
+description: HTML input element of type file is not working in the Android web view
+type: troubleshooting
+page_title: HTML File Input Not Responding in the Android web view
+slug: troubleshooting-input-type-file-not-working-in-android
+position:
+tags: appbuilder, android
+teampulseid:
+ticketid:  1002377,  1020566, 1018666
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS version</td>
+    <td>4.4 KitKat and above</td>	
+  </tr>  
+</table>
+
+## Description
+
+Using `<input type="file">`
+doesn't work on some Android devices. In most of the cases, when the **Choose File** button is clicked\tapped there is no response from the web view and the user cannot select a file from the device. 
+
+
+## Steps to Reproduce
+
+1. Create a Cordova application.
+1. Insert the `<input type="file">` element in the `<body>` of the *index.html* file.
+1. Build and Deploy the project to an Android 4.4 or higher device.
```

Comment:
deploy, unless you're referring to the command name. In this case, however, you will have to explain how to run the command.